src/extensions/agent_modules/cloud_communication/eg_mqtt_broker_client/inc/aduc/agent_module_interface_internal.h

```diff
@@ -4,34 +4,34 @@
 #include <du_agent_sdk/agent_module_interface.h> // ADUC_AGENT_CONTRACT_INFO
```

Comment:
is _internal.h only referenced from "src", if so, I'd put this header there, and not in "inc".

Reply:
good point. Thanks, tracking it for later